doc/sphinx/proof-engine/tactics.rst

```diff
@@ -105,6 +105,227 @@ bindings_list`` where ``bindings_list`` may be of two different forms:
+  pattern :n:`?` is used to complete the list.
+
+**need example for [= ... ]**
+
+* :n:`( @simple_intropattern , ... , @simple_intropattern )` - matches
+  a product over an inductive type :g:`I` with a single
+  constructor that itself has at least `n` arguments.  It performs a case
+  analysis over the hypothesis, as :n:`destruct` would, and applies the
+  patterns to the arguments of the constructor of :g:`I`.  Note that
+  that :n:`({+ p})` is equivalent to :n:`[{+ p}]`);
+* :n:`( @simple_intropattern & ... & @simple_intropattern )` - :n:`({+& p})`
+  is a shortcut for matching with
+  :n:`(p,( ... ,( ..., p ) ... ))`.  It expects the hypothesis to be a sequence of
+  right-associative binary inductive constructors such as :g:`conj` or
+  :g:`ex_intro`; for instance, a hypothesis with type
+  :g:`A /\(exists x, B /\ C /\ D)` can be introduced via pattern
+  :n:`(a & x & b & c & d)`;
+
+
+**need example for & ... &**
+
+**A /\\ (exists x, B /\\ C /\\ D) is not valid for (a & x & b & c & d)**
+
+* :n:`*` - introduces one or more quantified variables from the result
+* :n:`**` - introduces one or more quantified variables or hypotheses from the result until there are
+  no more quantified variables or implications (:g:`->`).  :g:`intros **` is equivalent
+  to :g:`intros`.
+
+* :n:`%term` - a pattern :n:`p{+ %term}` first applies :n:`{+ term}`
+  on the hypothesis to be introduced (as in :n:`apply {+, term}`) prior to the
+  application of the introduction pattern :n:`p`;
+
+  the on-the-fly application of lemmas: :n:`p{+ %term}` where :n:`p`
+  itself is not a pattern for on-the-fly application of lemmas (note:
+  syntax is in experimental stage)  **huh?**
+
+**need example, maybe simplify "intros [= -> %IHl]." from List.v?**
+
+**operconstr: coqtop is not accepting this, not sure why--or what the construct means**
+**is it experimental?**
+
+.. note::
+
+   In :n:`intros @intro_pattern_list`, if the last introduction pattern
+   is a disjunctive or conjunctive pattern
+   :n:`[{+| @intro_pattern_list}]`, the completion of :n:`@intro_pattern_list`
+   so that all the arguments of the i-th constructors of the corresponding
+   inductive type are introduced can be controlled with the following option:
+
+   .. flag:: Bracketing Last Introduction Pattern
+
+      Force completion, if needed, when the last introduction pattern is a
+      disjunctive or conjunctive pattern (on by default).
+
+.. note::
+
+   :n:`intros {+ p}` is not equivalent to :n:`intros p; ... ; intros p`
+   for the following reason: If one of the :n:`p` is a wildcard pattern, it
+   might succeed in the first case because the further hypotheses it
+   depends on are eventually erased too while it might fail in the second
+   case because of dependencies in hypotheses which are not yet
+   introduced (and a fortiori not yet erased).
+
+Examples:
+
+.. _intropattern_conj_ex:
+
+   .. example:: intro pattern for /\\
```

Comment:
The text on the right appears in the header for the example in the output.  Seems useful